compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -581,6 +581,7 @@ struct QuantizeSpecialActivation final : public luci::CircleNodeMutableVisitor<v
   {
     auto input = loco::must_cast<luci::CircleNode *>(node->input());
     auto i_qparam = input->quantparam();
+    assert(i_qparam);
```

Comment:
(optional) @jinevening I'm just worrying about dereferencing null pointer problem that might be checked by static analysis tool either. Someday, it would be better to make these kind of functions return bool or sth and replace `assert` with return statement.